### ISSUE TITLE
Ternary whitespace check ?

### ISSUE THREAD

**Issue by jrfnl** [WordPress-Coding-Standards/WordPress-Coding-Standards#988]
Looks like whitespace around ternary operators is currently not checked.

Should it be ?

Upstream this has been added to the `WhiteSpace.OperatorSpacing` sniff. The WPCS version of this sniff has not been synced with upstream for quite a long time.
Might be worth checking if the WP version can be removed and defer to the upstream version instead.

```php
$a = ($a > 10)?$a:10;
```

will yield no errors.

I would expect this to error on:
* no whitespace before "inline then" / ? - expected 1 space
* no whitespace after "inline then" / ? - expected 1 space
* no whitespace before "inline else" / : - expected 1 space
* no whitespace after "inline else" / : - expected 1 space

As a side-note, I'm also missing these notifications:
* no whitespace after the open parenthesis
* no whitespace before the closing parenthesis
